Andrea / exercise-1/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pagina prueba</title>
+    <link rel="stylesheet" href="stylesheet.css"
+</head>
+<body>
+    <header>
+       <nav class="top-menu">
+        <img class="logo" src="logo.png" alt="imagen">
+           <ul class="top-menu__links">
+               <li>
+                   <a href="#">features</a>
+               </li>
+               <li>
+                   <a href="#">templates</a>
+               </li>
+               <li>
+                   <a href="#">integrations</a>
+               </li>
+               <li>
+                   <a href="#">pricing</a>
+               </li>
+               <!-- Botones -->
+               <li>
+                   <a class="top-menu__button" href="#">sign in</a>
+               </li>
+               <li>
+                   <a class="top-menu__button" href="#">sign up</a>
+               </li>
+           </ul>
+       </nav>
+
+        <!-- Greetings -->
+        <section class="hero">
+            <h1>The magic of reading</h1>
+        </section>
+    </header>
+
+
+    <!-- Achievements -->
+    <section>
+        <ul class="section-menu">
+            <li>
+                <p><img src="icon.png" alt="icon">genres</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">buy</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">fanart</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">communities</p>
+            </li>
+    </section>
+
+    <main>
+        <div class="title">
+            <p><span class="title-span">read<br>more</span> open your eyes</p>
+        </div>
+        
+        <div class="body">
+            <!-- <section class="columns-format"> quitar esta clase y ponerla en body -->
+                <div class="body-img">
+                    <img src="side_img.jpg" alt="imagen">
+                </div>
+                <div class="body-content">
```

Comment:
Same suggestion about the class name including the word "body". Also, we need to make fixes on the layout because the columns look funky on larger resolutions 
![image](https://user-images.githubusercontent.com/71283270/101287068-f2a41000-37a2-11eb-97e9-36ad364a14b1.png)

Reply:
Follow this guide to make the columns resize and its content correctly https://css-tricks.com/snippets/css/a-guide-to-flexbox/